L1Trigger/CSCTriggerPrimitives/src/CSCMotherboardME11.cc

```diff
@@ -482,7 +480,7 @@ std::vector<CSCCorrelatedLCTDigi> CSCMotherboardME11::getLCTs1a()
 }
 
 
-bool CSCMotherboardME11::doesALCTCrossCLCT(CSCALCTDigi &a, CSCCLCTDigi &c, int me)
+bool CSCMotherboardME11::doesALCTCrossCLCT(const CSCALCTDigi &a, const CSCCLCTDigi &c, int me) const
```

Comment:
@fabiocos In this pull request I updated the argument list of certain functions, `object -> const object&`. Functions were made const were applicable. This function `doesALCTCrossCLCT` however is the only one where a `object& -> const object&`. So that may introduce a slight change. Though, there is no reason why this function should change the ALCT and CLCT.